package.json

```diff
@@ -44,7 +44,7 @@
     "build": "REACT_APP_API_URL='https://w6pkliozjh.execute-api.us-east-1.amazonaws.com/Stage' react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "lint": "eslint . --ext js,jsx",
+    "lint": "eslint src/ --ext js,jsx --ignore-pattern \"*.story.jsx\"",
```

Comment:
I think stories should generally follow our eslint rules too.
If actually fixing them right now would be a bit much, I think that's fine. You could suppress the few specific errors we have in those files, or even leave them for now, just as long as we still notice any new kinds of lint errors we add to the stories.

Reply:
I disabled checking stories because I got an error on lint:

`'@storybook/react' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies`

This error seems incorrect - @storybook/react dependency should definitely be in devDependencies. So I decided to disable checking of stories. Let me know if you would prefer a different approach. Thanks.

Reply:
Yeah, when I said "suppress", I meant just using `// eslint-disable-next-line import/no-extraneous-dependencies`.

Although now I did a quick search, and saw this: https://github.com/storybooks/storybook/issues/1992.
So, looks like the best option would be replacing this line with:
```
'import/no-extraneous-dependencies': [2, { devDependencies: ['**/*.story.jsx'] }],
```